test/inductor/test_torchinductor_opinfo.py

```diff
@@ -241,6 +241,7 @@ def format_op(op):
     "to_sparse": {f16, f32, f64},
     "pca_lowrank": {f32, f64},
     "svd_lowrank": {f32, f64},
+    "torch.ops.aten._efficient_attention_forward": {f16, bf16, f32},
```

Comment:
What are the failures here ? Can we fix them ?

Reply:
this op returns ints and it seems like every opinfo test hates that:
https://github.com/pytorch/pytorch/pull/110546

one resolution path